src/client/css/style.css

```diff
@@ -244,3 +264,93 @@ section#community{
   color: #FF0000;
   font-size: 10px;
 }
+/*---Download Section CSS---*/
+#download-section {
+  background-color:#565A5C;
+}
+#ff-download-message-h {
+  color:white;
+}
+#ff-download-btn, #ff-download-btn:hover, #ff-download-btn:visited, #ff-download-btn:active {
+  text-decoration: none !important;
+  color:white;
+}
+#ff-download-img {
+  margin:auto;
```

Comment:
Removing this causes the firefox download image to align to align to the right instead of aligning in the centre.